src/sinks/hdfs/mod.rs

```diff
@@ -0,0 +1,28 @@
+//! `hdfs` sink.
+//!
+//! This sink will send it's output to HDFS.
+//!
+//! `hdfs` is an OpenDal based services. This mod itself only provide
+//! config to build an [`OpenDalSink`]. All real implement are powered by
+//! [`OpenDalSink`].
+
+mod config;
+pub use self::config::HdfsConfig;
+
+#[cfg(test)]
+mod test;
+
+/// It's better to run this test inside container:
+///
+/// ```shell
+/// make test-integration-hdfs
```

Comment:
**note**: `cargo vdev integration test hdfs` would also run this 👍

Reply:
cargo vdev is ready cool!